app/views/preferences/index.html.erb

```diff
@@ -1,27 +1,38 @@
-<div id="display_box">
+<div class="bootstrap">
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-xs-12">
+        <div id="edit_error_status">
+          <%= get_list_of_error_messages_for(@user)%>
+          <%= get_list_of_error_messages_for(@prefs)%>
+        </div>
 
-  <div id="edit_error_status">
-    <%= get_list_of_error_messages_for(@user)%>
-    <%= get_list_of_error_messages_for(@prefs)%>
-  </div>
-
-  <%= form_for(@prefs) do %>
-  <div id="tabs">
-    <ul>
-      <li><a href="#tabs-1"><%= t('preferences.tabs.profile')%></a></li>
-      <li><a href="#tabs-2"><%= t('preferences.tabs.authentication')%></a></li>
-      <li><a href="#tabs-3"><%= t('preferences.tabs.date_and_time')%></a></li>
-      <li><a href="#tabs-4"><%= t('preferences.tabs.tracks_behavior')%></a></li>
-    </ul>
-    <div id="tabs-1"><%= render :partial => 'profile'%></div>
-    <div id="tabs-2"><%= render :partial => 'authentication'%></div>
-    <div id="tabs-3"><%= render :partial => 'date_and_time'%></div>
-    <div id="tabs-4"><%= render :partial => 'tracks_behavior'%></div>
-  </div>
+        <%= form_for(@prefs) do %>
+          <ul id="tabs" class="nav nav-tabs">
+            <li role="presentation" class="active">
+              <%= link_to t('preferences.tabs.profile'), "#profile", data: { toggle: "tab" } %>
+            </li>
+            <li role="presentation">
+              <%= link_to t('preferences.tabs.authentication'), "#authentication", data: { toggle: "tab" } %>
+            </li>
+            <li role="presentation">
+              <%= link_to t('preferences.tabs.date_and_time'), "#date_and_time", data: { toggle: "tab" } %>
+            </li>
+            <li role="presentation">
+              <%= link_to t('preferences.tabs.tracks_behavior'), "#behavior", data: { toggle: "tab" } %>
+            </li>
+          </ul>
+          <div class="tab-content">
+            <div role="tabpanel" class="tab-pane active" id="profile"><%= render :partial => 'profile'%></div>
```

Comment:
I know you're not touching this, so leave as-is if you like, but these `render`s don't need `:partial` to be a named parameter.